bundler/src/db/dao/transaction_dao.rs

```diff
@@ -107,6 +107,14 @@ impl TransactionDao {
             exponent: exponent?,
         })
     }
+
+    pub async fn update_user_transactions(&self, txn_id: String, txn_hash: String, status: String) {
+        let conn = connect(self.pool.clone()).await;
+        let mut stmt = conn
+            .prepare("UPDATE user_transactions set status=?, metadata = json_set(metadata, '$.transaction_hash', ?) where transaction_id=?")
```

Comment:
```suggestion
            .prepare("UPDATE user_transactions set status = ?, metadata = json_set(metadata, '$.transaction_hash', ?) where transaction_id = ?")
```

consistent spacing helps